deploy/db/sql/init/populate.sql

```diff
@@ -6,5 +6,14 @@ INSERT INTO notary.wallets (wallet, irohausername)
 VALUES ('0x00Bd138aBD70e2F00903268F3Db08f2D25677C9e', 'user2@notary');
 
 -- Populate tokens table
-INSERT INTO notary.tokens (wallet, token)
-VALUES ('0x00Bd138aBD70e2F00903268F3Db08f2D25677C9e', 'tkn');
+INSERT INTO notary.tokens (wallet, token) VALUES
```

Comment:
What about lines above, do we need them?

Reply:
Good point, I'll add task to remove PostgresWalletsProvider